### PR TITLE
Getter for viewInvMatrix property (for model viewer)

### PR DESCRIPTION
**Pull request by kpal81xd**
- Adds getter for `_viewInvMatrix` in XRView for model viewer (hidden from docs)

### CODE REVIEW

File: src/framework/xr/xr-view.js [hunk 1]
```diff
@@ -324,6 +324,14 @@ class XrView extends EventHandler {
         return this._viewInvOffMat;
     }
 
+    /**
+     * @type {Mat4}
+     * @ignore
+     */
+    get viewInvMat() {
```
Comment (willeastcott) [inline src/framework/xr/xr-view.js:331]: I know you've set this as `@ignore` but I don't like the name if this is intended as API. `invViewMat` is better. I've head the phrase 'inverse view matrix' before. I've never heard anyone say 'view inverse matrix'.

Comment (mvaligursky) [inline src/framework/xr/xr-view.js:331]: Camera component exposes viewMatrix only, I'd be keen to not expose this at all.

Comment (willeastcott) [reply src/framework/xr/xr-view.js:331]: Also, the naming isn't consistent with the `CameraComponent`'s `viewMatrix` and `projectionMatrix` properties.

Comment (mvaligursky) [reply src/framework/xr/xr-view.js]: Why is per-eye view matrix needed?

Comment (kpal81xd) [reply src/framework/xr/xr-view.js]: Okay sounds like this is a bigger refactor ill close this PR and create at a later date

Comment (kpal81xd) [reply src/framework/xr/xr-view.js]: It technically isnt but the model viewer gets the inverse matrix using `const mat = xr.views.list[0]._viewInvMat;`
but ive replaced it with `const mat = xr.camera.camera.viewMatrix.clone().invert();`

**Pull request closed without merging**